File: pset6/mario/mario.py
from cs50 import get_int


while True:
    n = get_int("Height: ")  # ask for number of line
    if n >= 0 and n <= 23:  # check input
        break
for i in range(n):  # go for every line
    for j in range(n - (i + 1)):  # print spcaes first
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")  # then #
    print("  ", end="")  # ten double space on the middle fo the tower
    for b in range(i + 1):
        print("#", end="")  # hashtag again
    print()  # new line

File: pset1/mario/mario.c
//towers of mario
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n = 0;
    do
    {
        n = get_int("Height:");
    }
    while (n < 0 || n > 23);

    for (int i = 0; i < n; i++) //line loop
    {
        for (int j = 0; j < n - (i + 1); j++) //space printing
        {
            printf(" ");
        }

        for (int k = 0; k < i + 1; k++) //# printing
        {
            printf("#");
        }

        printf ("  ");

        for (int b = 0; b < i + 1; b++) //# printing
        {
            printf("#");
        }
        printf("\n");
    }

}
